### ISSUE TITLE
KeyError: 'nextRequestSeconds'

### ISSUE THREAD

**Issue by mrakitin** [NSLS-II/sirepo-bluesky#24]
Noticed the issue in the [build log](https://dev.azure.com/nsls2/profile_collections/_build/results?buildId=2074&view=logs&j=01379a27-47fc-5e45-e3dd-439819cb52bf&t=7ed15aff-9b7d-5848-a770-25bbaaa714b9) for https://github.com/NSLS-II-TES/profile_sirepo/pull/4:

```py
Transient Scan ID: 156     Time: 2021-05-15 00:03:26
Persistent Unique Scan ID: '5a5cab26-4296-4148-8b14-640896890fd3'
copy ONlNwqU0, Youngs Double Slit Experiment Bluesky
copy e4aBb3Le, Youngs Double Slit Experiment Bluesky 2
running sim ONlNwqU0
running sim e4aBb3Le
Process Process-546:
Traceback (most recent call last):
  File "/home/vsts/miniconda/envs/collection-2021-1.2/lib/python3.7/multiprocessing/process.py", line 297, in _bootstrap
    self.run()
  File "/home/vsts/miniconda/envs/collection-2021-1.2/lib/python3.7/multiprocessing/process.py", line 99, in run
    self._target(*self._args, **self._kwargs)
  File "/home/vsts/miniconda/envs/collection-2021-1.2/lib/python3.7/site-packages/sirepo_bluesky/sirepo_flyer.py", line 372, in _run
    status = sim.run_simulation()
  File "/home/vsts/miniconda/envs/collection-2021-1.2/lib/python3.7/site-packages/sirepo_bluesky/sirepo_bluesky.py", line 165, in run_simulation
    time.sleep(res['nextRequestSeconds'])
KeyError: 'nextRequestSeconds'
New stream: 'sirepo_flyer'
...
```

Relevant piece of code: https://github.com/NSLS-II/sirepo-bluesky/blob/430396d2ee71e35912dca78044bbc1547e1becae/sirepo_bluesky/sirepo_bluesky.py#L165

**Comment by mrakitin** [comment]
@moellep / Paul, is it something that has been deprecated?

**Comment by moellep** [comment]
I verified that nextRequestSeconds is still being used in Sirepo. It looks like the test completed successfully yesterday:

https://dev.azure.com/nsls2/profile_collections/_build/results?buildId=2088&view=logs&j=8d99ad36-e016-5305-6a8b-ec3199d20735&t=fdef6bdd-c258-506f-09a8-681d9ef2fd71

The 2021-05-15 test run took 20 minutes to run before it failed. On 2021-05-17 it took 6 minutes and passed. I'm not sure if the first case hit a time limit. I'll add a pull request to sirepo_bluesky which prints more information if this failure happens again.

**Comment by mrakitin** [comment]
Thanks for looking into it, @moellep! Yes, the test where I observed it was a long-running job with many iterations (the default was 100, per [this default](https://github.com/NSLS-II/bloptools/blob/8e3f153deead45c2dcb79581b3743159455667df/bloptools/de_optimization.py#L289)). I reduced it to [5](https://github.com/NSLS-II-TES/profile_sirepo/blob/95cc226aaab2fa27a3d80b1207357bb6779bfff1/.ci/drop-in.py#L19) to allow for shorter integration tests.

The additional debug information will definitely help. I think printing the whole received response from the server would help. Then we can see how we can work it around (maybe retry or something similar?).
